src/App.css

```diff
@@ -1,33 +1,95 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 .App {
-  text-align: center;
+  max-width: 100%;
+  display: grid;
+  grid-template-rows: 50px 1fr;
+  grid-template-columns: 1fr;
+  width: 100vw;
+  height: 100vh;
+}
+
+.header {
+  display: flex;
+  background-color: #bfd3c1;
+  justify-content: center;
+  align-items: center;
+  position: sticky;
+  top: 0;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+.main-grid {
+  display: grid;
+  grid-template-rows: 80px 1fr;
+  grid-template-columns: 1fr;
+}
+.filter-section {
+  display: inline-flex;
+  flex-direction: row;
+  background-color: #68a691;
+  justify-content: space-between;
+  align-items: center;
+  overflow-x: auto;
+  white-space: nowrap;
+  width: 100vw;
+  position: sticky;
+  top: 50px;
+  z-index: 1;
+  box-shadow: 0px 3px 5px 0px rgba(87, 87, 87, 1);
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.filter-element {
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
   align-items: center;
+  border: 1px solid black;
+  padding: 5px;
+  overflow: scroll;
+  flex: 0 0 100px;
+  margin: 10px;
+  background-color: #e8ffea;
+}
+
+.list-section {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: scroll;
+  justify-content: center;
+  align-content: center;
+  background-color: #aebfb0;
+}
+
+.list-element {
+  display: flex;
+  background-color: #4e8053;
+  height: 150px;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  margin: 15px;
+  box-shadow: 2px 1px 1px 0px rgba(90, 90, 90, 0.75);
+}
+
+.image-left {
+  display: flex;
+  align-content: flex-start;
+  width: 100%;
+  height: 100%;
 }
 
-.App-link {
-  color: #61dafb;
+.img {
+  max-width: 100%;
```

Comment:
Is that necessary? Isn't it 100% by default?

Reply:
nope